packages/renderer/src/lib/extensions/ExtensionList.spec.ts

```diff
@@ -169,3 +170,45 @@ test('Expect to see empty screens on both pages', async () => {
   title = screen.getByText(`No extensions matching 'foo' found`);
   expect(title).toBeInTheDocument();
 });
+
+test('Search extension page searches also description', async () => {
+  catalogExtensionInfos.set([aFakeExtension]);
+  extensionInfos.set(combined);
+
+  render(ExtensionList, { searchTerm: 'bar' });
+
+  const myExtension1 = screen.getByRole('region', { name: 'idAInstalled' });
+  expect(myExtension1).toBeInTheDocument();
+
+  // second extension should not be there as only in catalog (not installed) and doesn't have "bar" in the description
+  const extensionIdB = screen.queryByRole('group', { name: 'B Extension' });
+  expect(extensionIdB).not.toBeInTheDocument();
+
+  cleanup();
+
+  // Change the search
+  render(ExtensionList, { searchTerm: 'foo' });
```

Comment:
Instead of using cleanup to _cleanup_ the `screen` you can directly use the output of the render function

```suggestion
  // Change the search
  const { queryByRole } = render(ExtensionList, { searchTerm: 'foo' });
  
  queryByRole(...)
```

Reply:
I couldn't get that to work - render() writes to some global document, and queryByRole finds leftover stuff from the beginning of the testt